hap.coffee

```diff
@@ -112,6 +120,20 @@ module.exports = (env) =>
         env.logger.debug("unsupported device type: " + device.constructor.name)
         return null
 
+    createAccessoriesFromTemplate: (device) =>
```

Comment:
Please make createAccessoriesFromTemplate the only method and remove createAccessoryFromTemplate. So in most cases the returned array will have just one item, but this way there is no duplicated code